harvester.go

```diff
@@ -66,11 +67,11 @@ func New(cfg interface{}) *Builder {
 }
 
 // WithConsulSeed enables support for seeding values with consul.
-func (b *Builder) WithConsulSeed(addr, dc, token string) *Builder {
+func (b *Builder) WithConsulSeed(addr, dc, token string, timeout time.Duration) *Builder {
```

Comment:
`dc` might be confusing as it is used mainly for `disconnect`. Could we use something else like `datacen` or `datactr` ?